viewer/packages/sector-loader/src/GltfSectorLoader.ts

```diff
@@ -35,6 +35,9 @@ export class GltfSectorLoader {
 
       const group = new AutoDisposeGroup();
 
+      // Note: This bounding box includes instanced meshes etc and may not be an accurate representation of the TriangleMesh bounding box.
+      const geometryBoundingBox = sector.metadata.geometryBoundingBox;
```

Comment:
Can we have a longer variable name, rather than a comment? E.g. `fullSectorBoundingBox`?